controller/common/customize/manager.go

```diff
@@ -116,7 +118,8 @@ func (rm *Manager) getRelatedClient(apiVersion, resource string) (*dynamicclient
 			klog.Warningf("related Manager %s cache sync never finished", rm.name)
 		}
 
-		rm.relatedInformers.Set(apiVersion, resource, informer)
+		groupVersion, _ := schema.ParseGroupVersion(apiVersion)
```

Comment:
Are we ok to ignore err (i.e. second response)?

Reply:
if something is wrong with `apiVersion`, it  would explode at line 103 (`informer, err = rm.dynInformers.Resource(apiVersion, resource)`